### ISSUE TITLE
the loss about Proxy loss

### ISSUE THREAD

**Issue by canteen-man** [littleredxh/DREML#8]
The Proxyloss in the loss.py.
The ”fvec“ multiply an transpose eye martix. So the value of the fvec is not change.Why multiply this?
And the proxy loss in this code is the softmax cross entropy.
But in the paper say use the 11th reference for loss.
The 11th refernce "No Fuss Distance Metric Learning using Proxies" is the loss about triplet loss,not the softmax cross entropy.
